classes/article/AuthorDAO.inc.php

```diff
@@ -120,9 +120,9 @@ function &getAuthorsAlphabetizedByJournal($journalId = null, $initial = null, $r
 				aa.middle_name,
 				aa.last_name,
 				CASE WHEN asl.setting_value = \'\' THEN NULL ELSE SUBSTRING(asl.setting_value FROM 1 FOR 255) END AS affiliation_l,
-				asl.locale,
+				CASE WHEN asl.setting_value = \'\' THEN NULL ELSE asl.locale END AS locale,
```

Comment:
I think this (and below) misses the case where asl.setting_value is NULL. It should be...

    CASE WHEN asl.setting_value = \'\' OR asl.setting_value IS NULL

Also below.